Python/stream_reader_sqlrace/atlas_session_writer.py

```diff
@@ -0,0 +1,286 @@
+import os
+import logging
+import struct
+import datetime
+from typing import List
+
+from ma.streaming.open_data.v1 import open_data_pb2
+from stream_reader_sqlrace.sql_race import SQLiteConnection
+
+logger = logging.getLogger(__name__)
+# configure pythonnet runtime for SQLRace API
+os.environ["PYTHONNET_RUNTIME"] = "coreclr"
+os.environ["PYTHONNET_CORECLR_RUNTIME_CONFIG"] = (
+    r"C:\Program Files\McLaren Applied Technologies\ATLAS 10\MAT.Atlas.Host.runtimeconfig.json"
```

Comment:
Why do we reference the A10 runtime config? Also, as you have an A10_INSTALL_PATH variable below, may be better to use that for the path to ensure they never get out of sync?

Reply:
The A10 runtime config is used to configure the .NET runtime in pythonnet, refactored to use the A10_INSTALL_PATH now.